DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -50,6 +50,14 @@ uint16_t buildTRDFeeID(int supermodule, int side, int endpoint)
   return feeid.word;
 }
 
+void buildTrackletMCMData(TrackletMCMData& trackletword, const uint slope, const uint pos, const uint q0, const uint q1, const uint q2)
+{
+  trackletword.slope = slope;
+  trackletword.pos = pos;
+  trackletword.pid = q0 + q1 & 0xff << 8; //q2 sits with a bit of q1 in the header pid word.
```

Comment:
I'm a bit rusty with operator precedence. Is this line the same as
`q0 | ( (q1 & 0xff) << 8 )`
I would find that more readable.

Reply:
The same yes, will bracket it.